### PR TITLE
Extract footer content

### PR DESCRIPTION
**Pull request by annekainicUSDS**
Completes https://github.com/usds/us-forms-system/issues/10

The footer content that is rendered on every form is very specific to Vets.gov. Abstract this out so instead `<FormApp>` renders whatever is passed to the form config under the property `footerContent`. This will enable users to create their own custom footer content component to render in their forms. I pulled out what had existed in `<FormApp>` for the footer and created a new component for it under `/platform` that will specific to only Vets.gov. I think added the `footerContent` property to all form configs across Vets.gov.

Also moved around a few other files that didn't belong under the shared schemaform code directory.

### CODE REVIEW

File: src/platform/forms/tests/components/FormFooter.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
```
Comment (jbalboni) [inline src/platform/forms/tests/components/FormFooter.unit.spec.jsx]: We've been trying to use Enzyme for new tests. It's not too different from skin deep: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/post-911-gib-status/tests/containers/IntroPage.unit.spec.jsx

File: src/applications/edu-benefits/1990/config/form.js [hunk 2]
```diff
@@ -102,6 +103,7 @@ const formConfig = {
   },
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
+  footerContent: FormFooter,
   getHelp: GetFormHelp,
```
Comment (jbalboni) [inline src/applications/edu-benefits/1990/config/form.js]: Seems like we could remove getHelp in these configs?

**Review by jbalboni — APPROVED** [review]
Couple comments, but overall LGTM.

File: src/platform/forms/tests/components/FormFooter.unit.spec.jsx (hunk 1 above)
Comment (annekainicUSDS) [reply src/platform/forms/tests/components/FormFooter.unit.spec.jsx]: I will convert these tests to Enzyme, thanks for catching that!

File: src/applications/edu-benefits/1990/config/form.js (hunk 2 above)
Comment (annekainicUSDS) [reply src/applications/edu-benefits/1990/config/form.js]: Hm, not sure I totally understand. The new `FormFooter` component checks if anything is specified for `getHelp` and if so renders what has been passed to it. Some forms don't include additional getHelp text in the config, so I think we still need to pass it to the config where we specifically want it.

Comment (jbalboni) [reply src/applications/edu-benefits/1990/config/form.js]: Oh, right, I see. I think we'll want to change that later; I'd rather just have a different component for forms that use GetHelp vs those that don't. You don't need to change that in here, though, I can make a ticket to refactor that later.

Comment (annekainicUSDS) [reply src/applications/edu-benefits/1990/config/form.js]: ah gotcha, that makes sense

**Pull request merged**